test/proto/BUILD

```diff
@@ -12,3 +12,29 @@ envoy_proto_library(
     name = "helloworld_proto",
     srcs = [":helloworld.proto"],
 )
+
+envoy_proto_library(
+    name = "bookstore_proto",
+    srcs = [":bookstore.proto"],
+    deps = [
+        "@googleapis//:http_api_protos",
+        "@protobuf_bzl//:cc_wkt_protos",
```

Comment:
Same point as earlier here and everywhere, we can't use "@foo" deps, they need to be bound to something that gets referenced in `external_deps`.